gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -118,6 +122,18 @@ def main(args):
         training_x_ht = training_dataset("chrx").ht()
         training_y_ht = training_dataset("chry").ht()
 
+        # Build models for sites on an autosome or a pseudoautosomal region, chromosome X, and chromosome Y
```

Comment:
I feel that we always analyze sex chromosome separately. It is because the AF and coverage distribution in sex chromosome is different?

Reply:
the sex chromosomes are biologically different from the autosomes, which is why we handle them differently

Reply:
```suggestion
        # Build plateau and coverage models for autosomes/pseudoautosomal regions, chromosome X, and chromosome Y
```